doc/source/reference/maskedarray.baseclass.rst

```diff
@@ -33,7 +33,7 @@ defines several constants.
 
    Value indicating that a masked array has no invalid entry.
    :attr:`nomask` is used internally to speed up computations when the mask
-   is not needed.
+   is not needed. It's represented internally as `np.False_`.
```

Comment:
```suggestion
   is not needed. It is represented internally as ``np.False_``.
```
Same as below, should have double ticks (I also do not like the abbreviations `it's` much, but maybe its just me, and I am a nonnative speaker (it does not bother me much either).

Anyway, as Ross says, this looks good to me, and I may just commit that change and merge it the next time I glance at this PR, thanks.